src/Integration.Vsix.UnitTests/SonarLintTagger/IssueToIssueMarkerConverterTests.cs

```diff
@@ -75,15 +75,16 @@ public void Convert_NullSpan_Null()
         public void Convert_SpanIsNotNull_ReturnsMarkerWithSpan()
         {
             var mockIssue = Mock.Of<IAnalysisIssue>();
-            var mockSnapshot = Mock.Of<ITextSnapshot>();
-            var mockSpan = new SnapshotSpan();
+            var mockSnapshot = new Mock<ITextSnapshot>();
+            mockSnapshot.SetupGet(x => x.Length).Returns(20);
```

Comment:
Need to create now a valid span, since `new SnapshotSpan()` is an empty one and will be considered invalid.